### PR TITLE
3.0.0

### PR DESCRIPTION
**Pull request by koji-1009**
Preparing for the final release.

### CODE REVIEW

**Review by m-yamada04 — APPROVED** [review]
LGTM!

**Review by nappannda — APPROVED** [review]
LGTM〜〜〜

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+3.0.0 Release notes (2021-03-29)
```
Comment (koji-1009) [inline CHANGELOG.md:1]:
```suggestion
3.0.0 Release notes (2021-03-31)
```

**Review by koishi — APPROVED** [review]
🙆🏼‍♂️

**Pull request merged**